src/protocol/boolean/prefix_or.rs

```diff
@@ -306,24 +306,29 @@ mod tests {
     use crate::{
         error::Error,
         ff::{Field, Fp2, Fp31},
-        protocol::{QueryId, RecordId},
+        protocol::{context::ProtocolContext, QueryId, RecordId},
         secret_sharing::Replicated,
-        test_fixture::{make_contexts, make_world, share, validate_and_reconstruct, TestWorld},
+        test_fixture::{
+            logging, make_contexts, make_world, share, validate_and_reconstruct, TestWorld,
+        },
     };
     use futures::future::try_join_all;
     use rand::distributions::{Distribution, Standard};
     use rand::{rngs::mock::StepRng, Rng};
     use std::iter::zip;
 
-    const BITS: [usize; 2] = [16, 32];
+    const BITS: [usize; 1] = [32];
```

Comment:
When I execute a test case for `PrefixOr` with different bit-lengths, the protocol halts during the first calls to `or()` at the second `BITS` loop. For example, if I have:

`const BITS: [usize; 2] = [16, 32];`

All cases `TEST_TRIES` times for 16-bit passes. But the first calls to `or()`, which is called from `step1()`, of 32-bit test cases stops indefinitely. How can I debug this?

[protocol__boolean__prefix_or__tests__fp31.log](https://github.com/private-attribution/ipa/files/10053935/protocol__boolean__prefix_or__tests__fp31.log)

Reply:
Resolved. `narrow`ing at the beginning of each loop before testing different bit-lengths solved the issue.